CONTRIBUTING.md

```diff
@@ -97,6 +97,17 @@ Nobody is perfect, and sometimes we mess things up. That said, here are some goo
 * Engage in excessive "feng shui programming" by moving code around without quantifiable benefit.
 * Add jokes or other "funny" things to user-facing parts of the system.
 
+## On usage of AI and LLMs
+
+Usage of AI to help in writing code is usually fine, but you are responsible for making sure the quality of the output
+is up to the standards of the project. Currently, AI-generated output is often too verbose and its quality is subpar
+compared to what most human contributors produce.
+
+Do not use an AI or LLM:
+
+* to generate changes without holding it to the same standards as human-written changes.
+* to respond to issues or PRs.
```

Comment:
Well, this feels kinda restricting to me. Not everyone is as good in English as you are, and sometimes it using ChatGPT to communicate feels like the right thing to do.
For example, the comment above is rewritten by ChatGPT as
> Well, this feels a bit restrictive to me. Not everyone is as good at English as you are, and sometimes using ChatGPT to communicate feels like the right choice.

For more complex explanations, it might have more significant difference. For example, take my this comment: https://github.com/LadybirdBrowser/ladybird/issues/2705#issuecomment-2512222347
If the original author used ChatGPT to process it, it would become way more readable, especially  for someone not familiar with Russian:
> To clarify, the prefixes were added by a standard PostCSS autoprefixer. Without them, the result appears to be unchanged.
Your assumptions about the error are valid. I'm not a browser developer, and I mean no offense. My intent is only to show exactly what isn't working.
Thank you for your hard work!

So, in my opinion, it should be ok to respond using LLM, but only for style/rewording/translation, but the general idea of the message should be definitely provided by a human..

Reply:
Maybe the "without holding it to the same standards as human-written ..." could cover both cases?

Reply:
Yes, it's definitely okay to use AI/LLMs for *assistance* when commenting on issues and PRs. The thing we want to avoid is people making automated responses with less-than-appropriate human involvement :)

Maybe the text can be refined to make this clear?

Reply:
I've reworded the section to reflect this.